packages/xo-server-sdn-controller/src/index.js

```diff
@@ -836,7 +836,7 @@ class SDNController extends EventEmitter {
       return
     }
 
-    this._newHosts = this._newHosts.slice(this._newHosts.indexOf(newHost), 1)
+    this._newHosts.splice(this._newHosts.indexOf(newHost), 1)
```

Comment:
You can merge `find` and `indexOf` by using `findIndex` which removes an iteration.